### PR TITLE
[RCF-1100] Unit Test Cases

### PR DESCRIPTION
**Pull request by SachinPremkumar**
[RCF-1100]

[RCF-1100]: https://mosip.atlassian.net/browse/RCF-1100?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.2.0'
+        classpath 'com.android.tools.build:gradle:7.2.2'
```
Comment (jainhitesh9998) [inline android/build.gradle:10]: any reason we have updated this?

File: android/build.gradle [hunk 2]
```diff
@@ -27,7 +27,7 @@ allprojects {
 }
 
 ext {
-    compileSdkVersion = 31
+    compileSdkVersion = 33
```
Comment (jainhitesh9998) [inline android/build.gradle:30]: reasoning behind min complile sdk version update?

Comment (SachinPremkumar) [reply android/build.gradle:30]: Test cases was not working for 31 so upgraded the compilesdkversion 33

File: android/build.gradle (hunk 1 above)
Comment (SachinPremkumar) [reply android/build.gradle:10]: Test cases was not working for 7.2.0 so upgraded the gradle version 7.2.2

**Review by jainhitesh9998 — APPROVED** [review]

**Pull request merged**